packages/strapi-design-system/src/MainNav/NavSections.js

```diff
@@ -1,18 +1,24 @@
 import React, { Children } from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 import { Stack } from '../Stack';
 import { Box } from '../Box';
 
+const BoxGrow = styled(Box)`
+  flex-grow: 1;
+  overflow-y: scroll;
+`;
```

Comment:
Should this be a `Flex` then? Since `shrink` exists as a prop I think we should/ could add `grow` too.

Reply:
adding grow to `Flex` sounds good 
but I don't think we need to use `Flex` here 
<img width="575" alt="image" src="https://user-images.githubusercontent.com/71838159/212906397-a199d638-88fe-4178-b85f-179827233fb9.png">
it causes other UI issues and demands adding other props to fix some of them (e.g. `align-items: start`) that I don't find useful there - wdyt?

Reply:
Right, my bad!